tests/Internal/CancellationQueueTest.php

```diff
@@ -96,7 +96,7 @@ public function rethrowsExceptionsThrownFromCancel(): void
         $cancellationQueue();
     }
 
-    private function getCancellableDeferred(): Deferred
+    private function getCancellableDeferred(): Deferred /** @phpstan-ignore-line */
```

Comment:
Is this still needed? Should probably include `@return Deferred<never>`.

Reply:
Included that fixes it.